mkt/websites/models.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+from django.conf import settings
+from django.db import models
+
+import json_field
+
+from mkt.site.models import ModelBase
+from mkt.translations.fields import TranslatedField
+from mkt.tags.models import Tag
+from mkt.translations.fields import save_signal
+
+
+class Website(ModelBase):
+    default_locale = models.CharField(max_length=10,
+                                      default=settings.LANGUAGE_CODE)
+    url = TranslatedField()
+    title = TranslatedField()
+    short_title = TranslatedField()
+    description = TranslatedField()
+    keywords = models.ManyToManyField(Tag)
+    # FIXME regions
```

Comment:
Copy `Webapp` mechanism (M2M like `AddonExcludedRegion`) ? Or use `json_field` like we do for categories ? It would be nice to re-use `RegionForm`.